bindings/nodejs/types/network.ts

```diff
@@ -24,16 +39,27 @@ export type Node = {
     disabled?: boolean;
 }
 
-export interface ClientOptions {
+interface NodeManagerBuilder {
     primaryNode?: string | Node;
     primaryPoWNode?: string | Node;
-    node?: string | Node;
     nodes?: Array<string | Node>;
+    permanodes?: Array<string | Node>;
     network?: string;
-    mqttBrokerOptions?: MqttBrokerOptions;
-    quorumSize?: number;
-    quorumThreshold?: number;
-    localPow?: boolean;
+    nodeSyncEnabled: boolean;
+    nodeSyncInterval: number;
+    quorum: boolean;
+    minQuorumSize: number;
+    quorumTreshold?: boolean;
+}
+
+export interface ClientOptions {
+    nodeManagerBuilder?: NodeManagerBuilder;
+    brokerOptions: MqttBrokerOptions;
+    networkInfo: NetworkInfo;
+    apiTimeout: number;
+    remotePowTimeout: number;
+    offline: boolean;
+    powWorkerCount: number;
```

Comment:
Here they're also all optional and not sure if this format works or is the wanted one, because one can(need to use?) the one where the structure if flatten

Reply:
I'm not sure if I understand the implications of this. Does this mean that I should extend the `ClienOptions` interface with all the properties from `NodeManagerBuilder` instead of having it as a separate type?

Reply:
Yes, unless it also works as it is currently (haven't tried it)

Reply:
No, it doesn't work. I updated it accordingly